doc/thirdpartypackages/index.rst

```diff
@@ -282,6 +282,14 @@ border, and logo.
 .. image:: /_static/mpl_template_example.png
     :height: 330px
 
+figpager
+=====
```

Comment:
undescores must span the element above.

Reply:
Thanks! I've extended them below.
[https://github.com/ebenp/matplotlib/commit/898e67f6517a0744a7c39596b757d15068b44f3d](https://github.com/ebenp/matplotlib/commit/898e67f6517a0744a7c39596b757d15068b44f3d)